### PR TITLE
Add first pass at instructions for github repo setup.

### PR DESCRIPTION
**Pull request by dbaron**
I wanted to document how we set up github repositories.  Here's a first pass at those instructions, though I think there's more to be added.

I poked around a bit to try to find something similar for w3c in general or for a few other groups, and was surprised I couldn't find one.  But perhaps I wasn't using the right search terms.

### CODE REVIEW

**Comment by dbaron** [comment]
Also, you can [see what the added markdown file looks like](https://github.com/dbaron/process/blob/github-setup/setting-up-github-repositories.md).

File: setting-up-github-repositories.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+These documents are generally maintained in either
+[bikeshed](https://github.com/tabatkins/bikeshed/)
+or
+[respec](https://github.com/w3c/respec/).
+
+Also see [W3C's FAQ on using GitHub](https://w3c.github.io/faq.html).
+
+## Structuring the repository
+
+The primary branch of the repository should be `master`, which is `git`'s default.  (We should not manually interact with a `gh-pages` branch, although some older repositories are set up to do so.)
```
Comment (hober) [inline setting-up-github-repositories.md:14]: I don't think we should recommend using the name `master` for the primary branch, despite it being `git`'s default.

**Comment by hober** [comment]
I've never used Travis before, so I had to sign up for an account and connect it to various GH orgs and repos. That feels weird to me; if other @w3ctag repos are already hooked up in Travis, do I personally really need to generate a new SSH key etc.?

**Comment by hober** [comment]
> do I personally really need to generate a new SSH key etc.?

Or can I just copy the `.travis.yml` and `deploy_key.enc` files from w3ctag/design-principles?

File: setting-up-github-repositories.md (hunk 1 above)
Comment (dbaron) [reply setting-up-github-repositories.md]: I hope I've addressed your concern in https://github.com/w3ctag/process/pull/5/commits/0430f1c168f2be4fcc5488a1671e8d273c24ed60 -- but perhaps you could double-check?

Comment (hober) [reply setting-up-github-repositories.md]: Looks good.

**Comment by dbaron** [comment]
I think you may just be able to copy it -- but note that the design-principles deploys happen under my account.  That's been happening since we reduced Domenic's (I think) access in 2017 and I had to redo the config under my account!

**Comment by hober** [comment]
I went ahead and generated a new key.

Note that I've had to install the Travis command line tool in order to follow Domenic's steps.

**Pull request merged**